src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -19056,9 +19075,9 @@ static void Test2()
                 // (21,26): warning CS8600: Converting null literal or possible null value to non-nullable type.
                 //                     p2 = null;
```

Comment:
Nit: Update warning comments.

Reply:
It seems to be up-to-date already.

Reply:
@jcouv as in there's 3 warnings but only 2 comments in the code about the warnings.

Reply:
Sorry, I'd misunderstood. I'll add that to upcoming PR. Thanks